Dockerfile

```diff
@@ -77,6 +77,7 @@ RUN cd /app \
     && pip install --no-cache -r requirements/local.txt
 
 COPY --from=superset-node /app/superset/static/assets /app/superset/static/assets
+COPY --from=superset-node /app/superset-frontend /app/superset-frontend
```

Comment:
The huge directory `node_modules` is in `superset-frontend` so I think it's not a good idea to copy this one.